parachain/pallets/incentivized-channel/src/inbound/mod.rs

```diff
@@ -106,3 +113,37 @@ decl_module! {
 		}
 	}
 }
+
+impl<T: Config> Module<T> {
+
+	fn handle_fee(amount: BalanceOf<T>, relayer: &T::AccountId) {
+		if amount.is_zero() {
+			return;
+		}
+
+		let imbalance = match T::Currency::withdraw(&T::SourceAccount::get(), amount, WithdrawReasons::TRANSFER, KeepAlive) {
+			Ok(imbalance) => imbalance,
+			Err(err) => {
+				debug::error!("Unable to withdraw from source account: {:?}", err);
```

Comment:
Heads up that `debug` becomes `log` in https://github.com/Snowfork/polkadot-ethereum/pull/331